### DISCUSSION TITLE
Using statement should not require a variable declaration

### DISCUSSION THREAD

**Discussion by bzaar** [dotnet/csharplang#7022] [Language Ideas] [answered]
I propose a small syntactic improvement for the using statement. Instead of having to write:

```csharp
using var _ = new PerfLogger("MyMethod");
```

This should suffice:

```csharp
using new PerfLogger("MyMethod"); 
```

Note that the using _block_ already supports this syntax:

```csharp
using (new PerfLogger("MyMethod"))
{
    // do stuff
}
```

I don't see why the using statement couldn't do the same.

**Comment by 333fred** [comment, marked as answer]
See https://github.com/dotnet/csharplang/issues/2235.
